awsplugins/beanstalk/beanstalk.go

```diff
@@ -12,8 +12,8 @@ import (
 	"encoding/json"
 	"io/ioutil"
 
+	"github.com/aws/aws-xray-sdk-go/internal/logger"
```

Comment:
Could we directly have a folder named `logger` under `aws-xray-sdk-go` folder instead of being a sub folder inside `internal`?

Reply:
I made it internal so users of xray can not access the internal logging functions (we don't want packages outside xray to be able to use our logger).